coursedata/level-defaults/nl.yaml

```
@@ -497,7 +497,7 @@
         De `<` kijkt of het eerste getal kleiner is dan de tweede. Bijvoorbeeld met `leeftijd < 12` kun je kijken of `leeftijd` kleiner is dan 12.
         Als je wil kijken of een getal kleiner of even groot is dan een ander getal, kun je `<=` gebruiken. Bijvoorbeeld `leeftijd <= 11`.
         De `>` kijkt of iets groter is dan het tweede getal. Bijvoorbeeld met `punten > 10` kun je kijken of `punten` groter is dan 10.
-        ALs je wil kijken of een getal groter of juist even groot is als een ander getal, door `>=` te gebruiken. Bijvoorbeeld `punten >= 11`.
+        Als je wil kijken of een getal groter of juist even groot is als een ander getal, kun je `>=` gebruiken. Bijvoorbeeld `punten >= 11`.
```

```suggestion
        Als je wil kijken of een getal groter of even groot is als een ander getal, kun je `>=` gebruiken. Bijvoorbeeld `punten >= 11`.
```

Ik vind die juist daar een beetje gek, alsof het een tegenstelling is.